lib/tasks/import_data.rake

```diff
@@ -49,7 +49,7 @@ task :import_employee_data => :environment do
       # Then the 'qualifications' column
       quals = row['qualification']&.split("\n") || []
       quals.each do |q|
-        next if q == 'None'
+        next if q == 'None' || q == 'Paediatric First Aid'
```

Comment:
This will skip the first aid qualification if it's in the 'qualifications' column, but I think at the moment we're importing it with the `skills_mapping` from the 'skills' column, so we probably need to remove it from there